### ISSUE TITLE
Documentation of details of e2ee and key cross signing

### ISSUE THREAD

**Issue by balert** [vector-im/element-web#16530]
With three years of experience with a private synapse/riot/element installation and very happy with the progress made during that time (thanks!), I am about to install a server for my company with 30k users. However, I still manage to get in situations when element claims it is "Unable to decrypt: ..." some messages. 

Therefore, I would really like to read more documentation about the details behind key backup in order to find out what I am doing wrong. I would like to verify that my key backup is in place and working correctly for example, maybe even be able to test drive it, just in case. 

Unfortunately I can't find too much documentation about the key backup, and even though the UI and UX is really getting great lately (thanks for all your work!) I still miss a bit more documentation about this. The best I could find so far is the "End-to-end encryption" paragraph in https://element.io/help which is a bit vague imho.

(My latest experience with "Unable to decrypt": My vague theory what might have caused it is that I might have logged out from Element too quickly after setting up key backup, and I assume Element was not yet finished uploading the keys to the server. Is this even possible? What else could've happened? Or does key backup only cover backup of encryption keys that have been used for encrypted messages written AFTER key backup was setup? I believe it's not a very uncommon use case, that I log in to Element as a new user, and skip key backup to have a look at Element first, write a few messages and later decide that i want to setup key backup (of course BEFORE EVER logging out, otherwise clearly the keys will be lost, no discussion about this). In this case I would assume that Element would also save the keys of those old messages too, right?)

tl;dr: I would really be happy to see more docs about the security and privacy settings (key backup, cross signing, e2ee key export/import). Maybe it's already there and I don't find it?

**Comment by jryans** [comment]
Unfortunately, it doesn't exist yet. I think this basically duplicates https://github.com/vector-im/element-web/issues/14240 and / or https://github.com/vector-im/element-web/issues/16162, so I think it makes sense to close this one.
